Sources/WalletConnectPush/Client/Dapp/DappPushClientFactory.swift

```diff
@@ -19,9 +19,9 @@ public struct DappPushClientFactory {
     static func create(metadata: AppMetadata, logger: ConsoleLogging, keyValueStorage: KeyValueStorage, keychainStorage: KeychainStorageProtocol, networkInteractor: NetworkInteracting, syncClient: SyncClient) -> DappPushClient {
         let kms = KeyManagementService(keychain: keychainStorage)
         let subscriptionStore: SyncStore<PushSubscription> = SyncStoreFactory.create(name: PushStorageIdntifiers.pushSubscription, syncClient: syncClient, storage: keyValueStorage)
-
+        let messagesStore = KeyedDatabase<PushMessageRecord>(storage: keyValueStorage, identifier: PushStorageIdntifiers.pushMessagesRecords)
         let subscriptionStoreDelegate = PushSubscriptionStoreDelegate(networkingInteractor: networkInteractor, kms: kms)
-        let pushStorage = PushStorage(subscriptionStore: subscriptionStore, subscriptionStoreDelegate: subscriptionStoreDelegate)
+        let pushStorage = PushStorage(subscriptionStore: subscriptionStore, messagesStore: messagesStore, subscriptionStoreDelegate: subscriptionStoreDelegate)
```

Comment:
actually dapp does not require sync as it will be deprecated and it's used only for integration tests but as you did it already we can leave it ;)